packages/components-angular/src/app/examples/checkbox-example-controlled.component.ts

```diff
@@ -0,0 +1,26 @@
+import { ChangeDetectionStrategy, Component } from '@angular/core';
+import { type CheckboxUpdateEventDetail } from '@porsche-design-system/components-angular';
+
+@Component({
+  selector: 'page-checkbox-example-controlled',
+  template: `
+    <p-checkbox
+      name="some-name"
+      label="Some Label"
+      value="some-value"
+      [checked]="state['some-name']"
+      (update)="onUpdate($event)"
+    />
+    <p-text>some-name: {{ state['some-name'] ? 'checked' : 'not checked' }}</p-text>
```

Comment:
```suggestion
    <p-text>some-name: {{ state['some-name'] }}</p-text>
```

I would suggest to show the current states value here